index.html

```diff
@@ -3,12 +3,60 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
     <script src="script.js" defer></script>
-    <title>Title</title>
+    <title>Jarver's Crypt Maze</title>
 </head>
 <body>
+    <h1>Jarver's Crypt</h1>
+    <div class="maze collision">
+        <div class="start_area">
+            <h2>Start</h2>
+            <img src="images/coffin.png" alt="a coffin">
+        </div>
+        <div class="line">
+            <div class="vertical line_one thick"></div>
+            <div class="horizontal line_two thick"></div>
+            <div class="vertical line_three thick"></div>
+            <div class="horizontal line_four thick"></div>
+            <div class="vertical line_five thick"></div>
+            <div class="horizontal line_six thin"></div>
+            <div class="vertical line_seven thin"></div>
+            <div class="horizontal line_eight thick"></div>
+            <div class="horizontal line_nine thick"></div>
```

Comment:
this line is appearing 1px south of where it should be ... but maybe a browser issue :D